src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse2.cs

```diff
@@ -198,11 +218,41 @@ public static class Sse2
         /// </summary>
         public static Vector128<double> CompareGreaterThan(Vector128<double> left, Vector128<double> right) => CompareGreaterThan(left, right);
 
+        /// <summary>
+        /// int _mm_comigt_sd (__m128d a, __m128d b)
+        /// </summary>
+        public static bool CompareGreaterThanOrderedScalar(Vector128<double> left, Vector128<double> right) => CompareGreaterThanOrderedScalar(left, right);
+
+        /// <summary>
+        /// int _mm_ucomigt_sd (__m128d a, __m128d b)
+        /// </summary>
+        public static bool CompareGreaterThanUnorderedScalar(Vector128<double> left, Vector128<double> right) => CompareGreaterThanUnorderedScalar(left, right);
+
+        /// <summary>
+        /// __m128d _mm_cmpgt_sd (__m128d a,  __m128d b)
+        /// </summary>
+        public static Vector128<double> CompareGreaterThanScalar(Vector128<double> left, Vector128<double> right) => CompareGreaterThanScalar(left, right);
+
```

Comment:
Did you miss [`int _mm_comineq_sd (__m128d a, __m128d b)`](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_comineq_sd&expand=2682,4689,1052,1162)?

Reply:
And `int _mm_ucomineq_sd (__m128d a, __m128d b)`.

Reply:
They are at https://github.com/dotnet/coreclr/pull/15341/files/b3a5840ff609dbd8212c631da659b0154917dfc4#diff-0c40c8a7f5df6b8bc03aef2dea8f0884R316

I ended up going with the `bool Compare*OrderedScalar` and `bool Compare*UnorderedScalar` naming convention for these operations.

Reply:
I see, thanks. But the comment seems incorrect `_mm_comine_sd` -> `_mm_comineq_sd`.